src/main/java/com/nutomic/syncthingandroid/syncthing/AppConfigReceiver.java

```diff
@@ -0,0 +1,117 @@
+package com.nutomic.syncthingandroid.syncthing;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.support.v4.app.NotificationCompat;
+
+import com.nutomic.syncthingandroid.R;
+import com.nutomic.syncthingandroid.activities.MainActivity;
+
+/**
+ * Broadcast-receiver to control and configure SyncThing remotely
+ *
+ * Created by sqrt-1764 on 25.03.16.
+ */
+public class AppConfigReceiver extends BroadcastReceiver {
+    private static final int ID_NOTIFICATION_BACKGROUND_ACTIVE = 2;
+
+    public static final String ACTION_START = "com.nutomic.syncthingandroid.action.ACTION_SYNCTHING_START";
+    public static final String ACTION_STOP  = "com.nutomic.syncthingandroid.action.ACTION_SYNCTHING_STOP";
+    public static final String ACTION_STOP2 = "com.nutomic.syncthingandroid.action.ACTION_SYNCTHING_STOP2";
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        switch (intent.getAction()) {
+            case ACTION_START:
+                SyncthingHelper.startSynchthingService(context);
+                break;
+
+            case ACTION_STOP:
+                SyncthingHelper.stopSynchtingService(context);
+                break;
+
+            case ACTION_STOP2:
+                SyncthingHelper.stopSynchtingService2(context);
+                break;
```

Comment:
Starting the service is a one-liner, so having an extra function is more (mental) overhead than it's worth.

And the stopping is only so long because of the notification, which we wouldn't need when stopping the service from any other place. But it seems like a good idea to put it in a seperate function (just not in a new class).